ArduCopter/Log.cpp

```diff
@@ -547,8 +582,23 @@ const struct LogStructure Copter::log_structure[] = {
 // @Field: aY: Target acceleration, Y-Axis
 // @Field: aZ: Target acceleration, Z-Axis
 
-    { LOG_GUIDEDTARGET_MSG, sizeof(log_GuidedTarget),
-      "GUID",  "QBfffbffffff",    "TimeUS,Type,pX,pY,pZ,Terrain,vX,vY,vZ,aX,aY,aZ", "s-mmm-nnnooo", "F-BBB-BBBBBB" , true },
+    { LOG_GUIDED_POSITION_TARGET_MSG, sizeof(log_Guided_Position_Target),
+      "GPOS",  "QBfffbffffff",    "TimeUS,Type,pX,pY,pZ,Terrain,vX,vY,vZ,aX,aY,aZ", "s-mmm-nnnooo", "F-BBB-BBBBBB" , true },
+
+// @LoggerMessage: GATT
+// @Description: Guided mode attitude target information
+// @Field: TimeUS: Time since system startup
+// @Field: Type: Type of guided mode
+// @Field: Roll: Target attitude, Roll
+// @Field: Pitch: Target attitude, Pitch
+// @Field: Yaw: Target attitude, Yaw
+// @Field: RollRate: Roll rate in centidegrees/s
+// @Field: PitchRate: Pitch rate in centidegrees/s
+// @Field: YawRate: TYaw rate in centidegrees/s
+// @Field: Thrust: Thrust -1 to 1 
```

Comment:
We need to add climb_rate here too since the same field can be either thrust (-1,1) or interpreted as climb_rate. Then add it into the logger field and as its own input to the Log_Write_Guided_Attitude() too.
```suggestion
// @Field: Thrust: Thrust -1 to 1 
// @Field: Climb Rate
```

Reply:
Done!